src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json

```diff
@@ -7,7 +7,7 @@
   }, {
     "tagName" : "cs2101",
     "fileAddress" : "./src/test/java/seedu/address/testutil/cs2101.bat",
-    "labels": ["testLabel"]
+    "labels": ["validLabel"]
```

Comment:
I'm not sure if you can change this only. You can refer to the rest of the file. I'm not sure why storage test can pass if you only changed this label but not the rest.

```
    "tagName" : "myfile 3",
    "fileAddress" : "c:\\a\\b\\myfile3.txt",
    "labels": ["testLabel"]
  }, {
    "tagName" : "myfile 4",
    "fileAddress" : "c:\\a\\b\\myfile4.txt",
    "labels": ["testLabel"]
  }, {
    "tagName" : "myfile 5",
    "fileAddress" : "c:\\a\\b\\myfile5.txt",
    "labels": ["testLabel"]
```

Reply:
Actually it will pass. But do you have a reason for changing this?

Reply:
Currently, in UnlabelCommandTest, I am using VALID_PATH that is in the CommandTestUtil. However, I need the VALID_PATH to be different with the default label in the TagBuilder, so that's why I am changing it.

Reply:
Actually, i'll try to change how the UnlabelCommand is tested. I'll revert back everything.

Reply:
Alrighty👍